aspnetcore/blazor/state-management.md

```diff
@@ -260,13 +579,13 @@ Prerendering might be useful for other pages that don't use `localStorage` or `s
 
     private async Task LoadStateAsync()
     {
-        currentCount = await ProtectedLocalStore.GetAsync<int>("prerenderedCount");
+        currentCount = await ProtectedLocalStore.GetAsync<int>("count");
```

Comment:
This should be updated to
```suggestion
        var result = await ProtectedLocalStore.GetAsync<int>("count");
        currentCount = result.Success ? result.Value : 0;
```
just like you have done elsewhere.

Same applies to other occurrences of `currentCount = await ...`

Reply:
Same thing ... this is in the <5.0 area and pertains to use of the experimental package.